packages/runtime/src/modules/AttributeListenerModule.ts

```diff
@@ -19,20 +21,68 @@ export class AttributeListenerModule extends RuntimeModule {
         const createdAttribute = event.data;
         if (createdAttribute.content["@type"] === "IdentityAttribute" && createdAttribute.shareInfo) return;
         if (createdAttribute.content["@type"] === "RelationshipAttribute" && createdAttribute.content.confidentiality === RelationshipAttributeConfidentiality.Private) return;
+        if (await AttributeListenerModule.relationshipAttributeOfInactiveRelationship(services, createdAttribute)) return;
 
         const getAttributeListenersResult = await services.consumptionServices.attributeListeners.getAttributeListeners();
         if (getAttributeListenersResult.isError) {
-            this.logger.error("Could not get attribute listeners", getAttributeListenersResult.error);
+            this.logger.error("Could not get LocalAttributeListeners", getAttributeListenersResult.error);
             return;
         }
 
         const attributeListeners = getAttributeListenersResult.value;
         const promises = attributeListeners.map((attributeListener) =>
-            this.createRequestIfAttributeMatchesQuery(services, attributeListener, event.data, event.eventTargetAddress)
+            this.createRequestIfAttributeMatchesQuery(services, attributeListener, createdAttribute, event.eventTargetAddress)
         );
         await Promise.all(promises);
     }
 
+    private static async relationshipAttributeOfInactiveRelationship(services: RuntimeServices, attribute: LocalAttributeDTO): Promise<boolean> {
```

Comment:
1. naming. a function returning an attribute should be named isBlaBlaBla
2. why static?

Reply:
2. I used `static` because I do not need to call a method or an attribute of an instance of the class `AttributeListenerModule` to call `relationshipAttributeOfInactiveRelationship`.
1. I will try to find a new wording that is coherent in combination with the prefix `AttributeListenerModule`, which is caused by the `static` function declaration currently used, as well.

Reply:
please don't use static here, It's way less readable on callee side.